apps/condo/bin/prepare.js

```diff
@@ -8,11 +8,16 @@ async function updateAppEnvAddressSuggestionConfig (serviceName) {
     await updateAppEnvFile(serviceName, 'FAKE_FINANCE_INFO_CLIENT', 'true')
 }
 
+async function updateAppEnvLocalSecrets (serviceName) {
+    await updateAppEnvFile(serviceName, 'ACCESS_TOKEN_SECRET_KEY', 'secret-key-very-looooooooooooong')
```

Comment:
if need to add a new variable it should be randomly generated! But really you can check during the code something like: if we didn't set this secret we should use the existing secret cookie secret.

And its better to use something like DATABASE_ENCRYPION_KEY